### PR TITLE
Change header description and card spacing

### PR DESCRIPTION
**Pull request by kojiadrianojr**
image
image

### CODE REVIEW

**Review by swengr-janan — APPROVED** [review]
Changed!

**Comment by kojiadrianojr** [comment]
Another update: 
image

File: src/views/Pools/components/ClaimAction/index.tsx [hunk 1]
```diff
@@ -11,19 +11,19 @@ const ClaimAction: React.FC = () => {
...
+          MGG
         </Text>
         <Text bold textTransform='uppercase' color='textSubtle' fontSize='12px'>
           Earned
         </Text>
       </ActionTitles>
       <ActionContent>
         <div>
-          <Earned>123</Earned>
+          <Earned>0</Earned>
```
Comment (medardm) [inline src/views/Pools/components/ClaimAction/index.tsx:22]: padynamic pls. su earned value man lang ini

**Review by medardm — CHANGES_REQUESTED** [review]

File: src/views/Pools/components/ClaimAction/index.tsx (hunk 1 above)
Comment (kojiadrianojr) [reply src/views/Pools/components/ClaimAction/index.tsx:22]: This is noted, pa approved nalang kuys ta isabay ko nalang etong change request duman sa fix modal na branch if okay lang?

Comment (kojiadrianojr) [reply src/views/Pools/components/ClaimAction/index.tsx:22]: This issue is for Banner ui update lang po sana, doon na ako sa modal fix na branch ma sabay kang dynamic data

Comment (kojiadrianojr) [reply src/views/Pools/components/ClaimAction/index.tsx:22]: I'll include this change request here kasi eto yung fix for modals. Is it okay? @medardm  
pr link -> https://github.com/mgguild/gamevault-frontend/pull/19

**Comment by kojiadrianojr** [comment]
Pending PR kuys @medardm  @swengr-janan  header update lang to baka pwede na ma AMD?

**Pull request closed without merging**